docs/markdown/guides/mermaid/styling.ts

```diff
@@ -0,0 +1,61 @@
+import {type MermaidConfig} from 'mermaid';
+
+/** Styling related mermaid configuration to use when generating mermaid charts. */
+export const mermaidStyling: MermaidConfig = {
+  themeCSS: `
+  background-color: var(--page-background) !important; // svg background color
```

Comment:
Does this CSS strictly need to be included as part of the pipeline, or could it just be included directly (perhaps as a Sass mixin) into the docs site? 

If not, could this live in a `.css` file that we import into this file?

Reply:
It likely could, I just ported this from the previous version.

Reply:
Done